### PR TITLE
Create Add authorized users in the RackConnect Management Interface

### LINKED ISSUE #1270: On boarding team doc suggestion - RackConnect Control Panel set up article needed
#Relates to #1264 
SME: Tyler Lutz
Input table: https://one.rackspace.com/display/devdoc/On+boarding+team+doc+suggestions
There doesn’t seem to be any docs on the RackConnect process, especially for MI customers.
The control panel set ups can be daunting for a customer.

### PR DESCRIPTION
**Pull request by kylelaffoon**
- Fixes #1270
Relates to #1315

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-03601b4e14/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-86e64ca928/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-86dd66dcd9/how-to/)

**Comment by kylelaffoon** [comment]
Will build "Add users in Cloud Sites Control Panel" article instead of this.

File: content/rackconnect/Add authorized users in the RackConnect Management Interface.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+and create or deactivate users.
+
+###Create a new user
+
+Complete the following steps to create and authorize new user.
+
+1. Click **Create New User**.
+2. In the **Users/New** screen, enter information for the new user in the
+   **First Name**, **Last Name**, **Email Address** and **Phone Number** fields and click **Create New User** when finished.
+   **Note:** This will generate an email to the new user to register. ???
```
Comment (cyrichardson) [inline content/rackconnect/Add authorized users in the RackConnect Management Interface.md]: Do you want ??? in there?

File: content/rackconnect/Add authorized users in the RackConnect Management Interface.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+permalink: add-authorized-users-in-the-rackconnect-management-interface/
+audit_date:
+title: Add authorized users in the RackConnect Management Interface
+type: article
+created_date: '2016-08-05'
```
Comment (cyrichardson) [inline content/rackconnect/Add authorized users in the RackConnect Management Interface.md]: Do you want to update the create and modified dates to today's date?

File: content/rackconnect/Add authorized users in the RackConnect Management Interface.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   **Note:** This will generate an email to the new user to register. ???
+3. After the new user is registered, open the  **Account > Permissions** page
+   to assign authorization for the user.
+   **Note:** If you already have a user whose authorization profile would be suitable for the user currently being assigned authorities, you can use the **Clone** option from the *User List** page.
+4. On the **Permissions / Assign By User** page, click on the name of the user
+   to assign authorization.
+   **Note:** Review the [Permission Guide](https://racker.my.rackspace.com/portal/downloads/csx-docs/PermissionsGuide.pdf), linked on the **Permissions / Assign By User** for information on assignment and permission management options.
+5. Under the **Account Permissions** tab, select the checkboxes for the
+   authorizations to assign.
+   **Note:** Assigning Account Administrator authorization will grant the same permissions as you have. If you don't want to assign that much authority, but are setting up a technical contact, be sure to assign at least the following:
```
Comment (cyrichardson) [inline content/rackconnect/Add authorized users in the RackConnect Management Interface.md]: will grant to grants

**Comment by cyrichardson** [comment]
@kylelaffoon Please see the inline comments and let me know when you are ready for me to review/merge.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e6bbd19525/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3366a99573/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5a94fa443c/how-to/).

**Comment by kylelaffoon** [comment]
@cyrichardson Applied your suggested changes. Thanks!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-61a1d6f8e0/how-to/).

**Comment by kylelaffoon** [comment]
@cyrichardson I recieved and implemented the review findings of the On boarding team. Please review and merge this PR at your convenience.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-519de89ffd/how-to/)

**Comment by cyrichardson** [comment]
@kylelaffoon I wonder if it's the \ at the end of the first line in the metadata (at the beginning of the article)?

**Comment by cyrichardson** [comment]
@kylelaffoon I don't see any error messages from the build. Do you?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4502577caa/how-to/)

**Comment by cyrichardson** [comment]
LGTM. Merging.

**Comment by kylelaffoon** [comment]
@cyrichardson You're awesome!

**Pull request merged**